src/sentry/utils/prompts.py

```diff
@@ -11,6 +11,7 @@
     "stacktrace_link": {"required_fields": ["organization_id", "project_id"]},
     "distributed_tracing": {"required_fields": ["organization_id", "project_id"]},
     "quick_trace_missing": {"required_fields": ["organization_id", "project_id"]},
+    "code_owners": {"required_fields": ["organization_id", "project_id"]},
```

Comment:
Is this descriptive enough?

Reply:
@EvanPurkhiser hmm I think its fine, bc we are calling the feature: Code Owners. Are you thinking like `suggest_upload_code_owners`?

Reply:
hmm, actually nah I think you're right it should be fine.